types/react-transition-group/CSSTransition.d.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from "react";
+import { TransitionActions } from "react-transition-group";
+import { TransitionProps } from "react-transition-group/Transition";
+
+export interface CSSTransitionClassNames {
+    appear?: string;
+    appearActive?: string;
+    enter?: string;
+    enterActive?: string;
+    exit?: string;
+    exitActive?: string;
+}
+
+export interface CSSTransitionProps extends TransitionProps {
+    classNames: string | CSSTransitionClassNames;
```

Comment:
Can probably copy the doc comment at https://github.com/reactjs/react-transition-group/blob/v2.0.2/src/CSSTransition.js#L12-L39 (minus the `@type`)